test-tools/wamr-ide/WASM-Debug-Server/Docker/README.md

```diff
@@ -3,7 +3,6 @@
 -   Linux
 
     ```shell
-    chmod +x resource/*
```

Comment:
Sometimes the files may lost executable permissions (e.g. copied from windows), had better add a notice that 
> Note: Ensure the scripts under `resource/scripts` and binaries under `resource/debug/bin` has executable permission

Reply:
ok. i added a note.